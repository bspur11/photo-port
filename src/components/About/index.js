import React from 'react';

import coverImage from "../../assets/cover/cover-image.jpg";

function About() {
  return (
   
    <section className="my-5">
       <img src={coverImage} className="my-2" style={{ width: "100%" }} alt="cover" />
      <h1 id="about">Who am I?</h1>
    </section>
   
  )}

export default About;