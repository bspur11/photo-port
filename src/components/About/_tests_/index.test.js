import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';

afterEach(cleanup);

describe('About component', () => {

  // first test 20.2.4
  it('renders', () => {
    render(<About />);
  });

  // second test
it('matches snapshot DOM mode structure', () => {
  // render About
  const { asFragment } = render(<About />);
  expect(asFragment()).toMatchSnapshot();
})
})

