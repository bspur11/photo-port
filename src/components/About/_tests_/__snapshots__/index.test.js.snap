// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM mode structure 1`] = `
<DocumentFragment>
  <section
    class="my-5"
  >
    <img
      alt="cover"
      class="my-2"
      src="cover-image.jpg"
      style="width: 100%;"
    />
    <h1
      id="about"
    >
      Who am I?
    </h1>
  </section>
</DocumentFragment>
`;
